sound/soc/intel/boards/Kconfig

```diff
@@ -419,7 +420,7 @@ config SND_SOC_INTEL_GLK_DA7219_MAX98357A_MACH
 	tristate "GLK with DA7219 and MAX98357A in I2S Mode"
 	depends on I2C && ACPI
 	depends on MFD_INTEL_LPSS || COMPILE_TEST
-	depends on SND_HDA_CODEC_HDMI
+	depends on SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC
```

Comment:
Here I'd just change the surrounding ifdef from SND_SOC_SOF_HDA_LINK to SND_SOC_SOF_HDA_AUDIO_CODEC. If all the GLk machine drivers need SOF and HDA codec drivers, then why not just have it in the ifdef.

Reply:
OK